PROJECT

```diff
@@ -1,7 +1,7 @@
-domain: deployment.skyscanner.net
-repo: github.com/Skyscanner/argocd-progressive-rollout
+domain: argoproj.skyscanner.net
```

Comment:
why this one?

Reply:
I added a note in the PR description. This is to:
- have consistency between our open source controllers. In the kms-issuer we have `cert-manager.skyscanner.net` as API group, here I just used the AppSet api group.
- if we ever build another controller targeting something in the argo ecosystem we can have the same API Group with a different CRD.

Note that if the project gets incubated we will probably need to change this again :)

Reply:
no worries, apologies I missed your note on this 👍